src/pages/_app.tsx

```diff
@@ -18,6 +22,12 @@ import EventsConfigProvider from '../common-events/configProvider/ConfigProvider
 import { useEventsApolloClient } from '../domain/clients/eventsApolloClient';
 import useEventsConfig from '../hooks/useEventsConfig';
 
+const matomoInstance = createMatomoInstance({
+  disabled: process.env.NEXT_PUBLIC_MATOMO_ENABLED !== 'true',
+  urlBase: process.env.NEXT_PUBLIC_MATOMO_URL_BASE as string,
+  siteId: Number(process.env.NEXT_PUBLIC_MATOMO_SITE_ID),
+});
```

Comment:
a matomo configuration from appConfig should be used instead.

Reply:
yes